<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NeedleDialog</class>
 <widget class="QDialog" name="NeedleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remove Needles and Cap</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="Box1">
     <property name="toolTip">
      <string>Threshold in degrees</string>
     </property>
     <property name="title">
      <string>Cap threshold (max. angle in degrees allowed within a triangle): </string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QDoubleSpinBox" name="capBox">
          <property name="toolTip">
           <string>Angle in degrees</string>
          </property>
          <property name="suffix">
           <string/>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="maximum">
           <double>360.000000000000000</double>
          </property>
          <property name="value">
           <double>160.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="Box2">
     <property name="toolTip">
      <string>(size+big edge )/(size+small edge)</string>
     </property>
     <property name="title">
      <string>Needle threshold (max. edge length ratio (longest/shortest) allowed within a triangle):</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QDoubleSpinBox" name="needleBox">
          <property name="toolTip">
           <string>Length Ratio</string>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
          <property name="value">
           <double>4.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="Box3">
     <property name="toolTip">
      <string>Do not collapse edges begger than this threshold.</string>
     </property>
     <property name="title">
      <string>Do not collapse needle edge larger than:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QDoubleSpinBox" name="collapseBox">
          <property name="toolTip">
           <string>Length</string>
          </property>
          <property name="decimals">
           <number>25</number>
          </property>
          <property name="maximum">
           <double>99999999.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="Box4">
     <property name="toolTip">
      <string>Do not collapse edges begger than this threshold.</string>
     </property>
     <property name="title">
      <string>Do not flip cap edge (according to the criterion above) if height is greater than:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QDoubleSpinBox" name="flipBox">
          <property name="toolTip">
           <string>Length</string>
          </property>
          <property name="decimals">
           <number>25</number>
          </property>
          <property name="maximum">
           <double>99999999.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NeedleDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NeedleDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
